packages/app-project/src/screens/ProjectHomePage/components/Hero/components/WorkflowSelector/WorkflowSelector.js

```diff
@@ -0,0 +1,70 @@
+import asyncStates from '@zooniverse/async-states'
+import { SpacedText } from '@zooniverse/react-components'
+import counterpart from 'counterpart'
+import { Box, Text } from 'grommet'
+import { arrayOf, shape, string } from 'prop-types'
+import React from 'react'
+import { withTheme } from 'styled-components'
+import { Bars } from 'svg-loaders-react'
+
+import WorkflowSelectButton from './components/WorkflowSelectButton'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function WorkflowSelector (props) {
+  const { workflows } = props
+  const loaderColor = props.theme.global.colors.brand
+
+  return (
+    <Box>
+      <SpacedText weight='bold' margin={{ bottom: 'xsmall' }}>
+        {counterpart('WorkflowSelector.classify')}
+      </SpacedText>
+      <Text>
+        {counterpart('WorkflowSelector.message')}
+      </Text>
+
+      {(workflows.loading === asyncStates.error) && (
+        <Box
+          align='center'
+          justify='center'
+          margin={{ top: 'small' }}
+        >
+          There was an error fetching the workflows :(
+        </Box>
+      )}
+
+      {(workflows.loading === asyncStates.success) && (
+        <Box margin={{ top: 'small' }}>
+          {workflows.data.map(workflow =>
+            <WorkflowSelectButton key={workflow.id} workflow={workflow} />
```

Comment:
Will the design variants for a couple and a lot of workflow button links come in separate PRs?